src/ctim/schemas/judgement.cljc

```diff
@@ -47,7 +47,9 @@
    (f/entry :priority Priority)
    (f/entry :confidence v/HighMedLow)
    (f/entry :severity v/HighMedLow)
-   (f/entry :valid_time c/ValidTime))
+   (f/entry :valid_time c/ValidTime)
+   (f/entry :title c/ShortString)
+   (f/entry :short_description c/MedString)))
```

Comment:
Not sure we want a `title` and the `description` because the `reason` field is here to explain the reason for issuing the judgement, or to provide additional context.

Reply:
I agree. This is a request from Craig and I'm not sure it's actually needed at all. The UI wants to put a title and description in for judgements, but I'm not really sure what those would be.